serverMod.js

```diff
@@ -67,29 +56,67 @@ module.exports.initMod = function (io, gameState, DATA) {
         var msg = "";
 
         if (global.consonant == 0) //a
-            msg = "What is art?";
+            msg = "Thank you for coming to the show";
         if (global.consonant == 1) //e
             msg = "Welcome to CCI diploma show";
         if (global.consonant == 2) //i
             msg = "Enjoy the show";
         if (global.consonant == 3) //o
-            msg = "Click on the different projects, to find out the info";
+            msg = "Click on the different projects";
         if (global.consonant == 4) //u
-            msg = "The play button will take you to the video presentations";
+            msg = "Play different presentations";
         if (global.consonant == 5) //none
             msg = "Have fun!";
 
 
         //sends a message to the room
         io.to("securityRoom").emit('nonPlayerTalked', { id: "", labelColor: "#3e7eb0", room: "securityRoom", message: msg, x: 36, y: 56 });
 
-    }, 60 * 1000); //every minute changes
+    }, 5 * 1000); //every ten seconds changes
```

Comment:
```suggestion
    }, 2.5 * 1000); //every 2.5 seconds changes
```

I feel like 5 seconds is a little slow